src/rgw/rgw_reshard.cc

```diff
@@ -993,6 +993,9 @@ int RGWReshard::process_single_logshard(int logshard_num)
       logshard_oid << dendl;
     return ret;
   }
+  if (ret < 0) {
+    return ret;
+  }
```

Comment:
I think we should just replace the `ret == -EBUSY` above with `ret < 0`

Reply:
And maybe also indicate the error code in the logged message, so we can better understand future failures.